cloudigrade/account/tests/test_views.py

```diff
@@ -944,3 +946,212 @@ def test_list_accounts_success(self, mock_get_primary_account_id):
+            self.account3,
+            is_encrypted=False,
+            is_windows=False,
+            ec2_ami_id=None,
+            is_rhel=False,
+            is_openshift=True)
+        self.openshift_and_rhel_image = account_helper.generate_aws_image(
+            self.account4,
+            is_encrypted=False,
+            is_windows=False,
+            ec2_ami_id=None,
+            is_rhel=True,
+            is_openshift=True)
+        self.event1 = \
+            account_helper.generate_single_aws_instance_event(
+                instance=self.instance1, powered_time=powered_time,
+                event_type=InstanceEvent.TYPE.power_on,
+                ec2_ami_id=self.windows_image.ec2_ami_id)
+        self.event2 = \
+            account_helper.generate_single_aws_instance_event(
+                instance=self.instance2, powered_time=powered_time,
+                event_type=InstanceEvent.TYPE.power_on,
+                ec2_ami_id=self.rhel_image.ec2_ami_id)
+        self.event3 = \
+            account_helper.generate_single_aws_instance_event(
+                instance=self.instance3, powered_time=powered_time,
+                event_type=InstanceEvent.TYPE.power_on,
+                ec2_ami_id=self.openshift_image.ec2_ami_id)
+        self.event4 = \
+            account_helper.generate_single_aws_instance_event(
+                instance=self.instance4, powered_time=powered_time,
+                event_type=InstanceEvent.TYPE.power_on,
+                ec2_ami_id=self.openshift_and_rhel_image.ec2_ami_id)
+        self.factory = APIRequestFactory()
+        self.account1_expected_overview = \
+            {'id': self.account1.aws_account_id,
+             'user_id': self.account1.user_id,
+             'type': 'aws',
+             'arn': self.account1.account_arn,
+             'creation_date': self.account1.created_at,
+             'name': self.account1.name,
+             'images': 1,
+             'instances': 1,
+             'rhel_instances': 0,
+             'openshift_instances': 0}
+        self.account2_expected_overview = \
+            {'id': self.account2.aws_account_id,
+             'user_id': self.account2.user_id,
+             'type': 'aws',
+             'arn': self.account2.account_arn,
+             'creation_date': self.account2.created_at,
+             'name': self.account2.name,
+             'images': 1,
+             'instances': 1,
+             'rhel_instances': 1,
+             'openshift_instances': 0}
+        self.account3_expected_overview = \
+            {'id': self.account3.aws_account_id,
+             'user_id': self.account3.user_id,
+             'type': 'aws',
+             'arn': self.account3.account_arn,
+             'creation_date': self.account3.created_at,
+             'name': self.account3.name,
+             'images': 1,
+             'instances': 1,
+             'rhel_instances': 0,
+             'openshift_instances': 1}
+        self.account4_expected_overview = \
```

Comment:
This is a minor style comment, but if you put the open curly brace on this line, you don't need to escape this newline.